src/wallet/components/WalletDropdown.tsx

```diff
@@ -35,33 +51,28 @@ export function WalletDropdown({ children, className }: WalletDropdownReact) {
     return null;
   }
 
-  if (breakpoint === 'sm') {
-    return (
-      <WalletBottomSheet className={className}>{children}</WalletBottomSheet>
-    );
+  if (!isSubComponentOpen) {
+    return null;
   }
 
   return (
-    <div
-      className={cn(
-        pressable.default,
-        color.foreground,
-        zIndex.dropdown,
-        'absolute right-0 mt-1.5 flex w-max min-w-[300px] cursor-default flex-col overflow-hidden rounded-xl',
-        isSubComponentClosing
-          ? 'fade-out slide-out-to-top-1.5 animate-out fill-mode-forwards ease-in-out'
-          : 'fade-in slide-in-from-top-1.5 animate-in duration-300 ease-out',
-        className,
-      )}
-      onAnimationEnd={() => {
-        if (isSubComponentClosing) {
-          setIsSubComponentOpen(false);
-          setIsSubComponentClosing(false);
-        }
-      }}
-      data-testid="ockWalletDropdown"
-    >
-      {childrenArray}
-    </div>
+    <WalletAdvancedProvider>
+      <div
+        data-testid="ockWalletDropdown"
+        className={cn(
+          'absolute',
+          showSubComponentAbove ? 'bottom-full' : 'top-full',
+          alignSubComponentRight ? 'right-0' : 'left-0',
+          className,
+        )}
+      >
+        <WalletAdvancedContent
```

Comment:
one nice thing we get "for free" with this refactor is that all dropdowns get the positioning logic. 

right now, if you create a draggable "non-advanced" wallet, you get this:
![image](https://github.com/user-attachments/assets/d7192aa3-0ecf-4479-84f3-32bdf120ea46)

but with this change, you get this:
![image](https://github.com/user-attachments/assets/061a57b3-f4d0-4caf-9f99-48e8399c9bc1)